### ISSUE TITLE
Beta V4.10821: Dashboard no updates on toggle scene/switch.

### ISSUE THREAD

**Issue by rwaaren** [domoticz/domoticz#3264]
Toggling switches/scenes on the dashboard tab **do not** update the webpage. 
Toggling switches/scenes in scenes or switches tab **do** update the webpage.

Any **first** toggle in fact does the update eg switching on and off. It is just the dashboard display that is not showing actual status. And as the dashboard is not updating you cannot switch off (or on).

V4.10717: 
-- repeating switch commands (pressing the icon repeatedly)
http://pi-2:8080/json.htm?type=command&param=switchlight&idx=1&switchcmd=Off&level=0&passcode=
http://pi-2:8080/json.htm?type=command&param=switchlight&idx=1&switchcmd=On&level=0&passcode=
http://pi-2:8080/json.htm?type=command&param=switchlight&idx=1&switchcmd=Off&level=0&passcode=
http://pi-2:8080/json.htm?type=command&param=switchlight&idx=1&switchcmd=On&level=0&passcode=

-- automatic screen refreshes
http://pi-2:8080/json.htm?type=devices&filter=all&used=true&favorite=1&order=[Order]&plan=0&lastupdate=1558677344

result: Dashboard changes. Repeated switching toggles switch 

V4.10821
-- repeating switch commands (pressing the icon repeatedly)
http://PI-1:8084/json.htm?type=command&param=switchlight&idx=249&switchcmd=On&level=0&passcode=
http://PI-1:8084/json.htm?type=command&param=switchlight&idx=249&switchcmd=On&level=0&passcode=
http://PI-1:8084/json.htm?type=command&param=switchlight&idx=249&switchcmd=On&level=0&passcode=
http://PI-1:8084/json.htm?type=command&param=switchlight&idx=249&switchcmd=On&level=0&passcode=
http://PI-1:8084/json.htm?type=command&param=switchlight&idx=249&switchcmd=On&level=0&passcode=

-- automatic screen refreshes
http://PI-1:8084/json.htm?type=devices&filter=all&used=true&favorite=1&order=[Order]&plan=0&lastupdate=1558677413

Console in Chrome developer tools window:

Uncaught Error: Syntax error, unrecognized expression: [object Object] #bstatus
    at Function.oe.error (jquery-3.3.1.min.js:2)
    at oe.tokenize (jquery-3.3.1.min.js:2)
    at oe.select (jquery-3.3.1.min.js:2)
    at Function.oe [as find] (jquery-3.3.1.min.js:2)
    at w.fn.init.find (jquery-3.3.1.min.js:2)
    at new w.fn.init (jquery-3.3.1.min.js:2)
    at w (jquery-3.3.1.min.js:2)
    at Object.<anonymous> (DashboardController.js:907)
    at Function.each (jquery-3.3.1.min.js:2)
    at Object.success (DashboardController.js:202)

result: Dashboard does not change. Repeated switching does nothing. 
forced refresh of screen [ CTRL r ] in Chrome) shows correct screen but only once. So after every switch command a [ CTRL r ] is required
to get a functioning dashboard.
 
[Forum link](https://www.domoticz.com/forum/viewtopic.php?f=6&t=28092)

**Comment by gizmocuz** [comment]
Could be an issue caused by a recent change here.... 
Seems the error is on line 907 of the DashboardController.js

**Comment by gizmocuz** [comment]
Found it, thanks for reporting!

**Comment by rwaaren** [comment]
Seems to be back / never gone ? in V4.10840

jquery-3.3.1.min.js:2 Uncaught Error: Syntax error, unrecognized expression: [object Object] #bstatus
    at Function.oe.error (jquery-3.3.1.min.js:2)
    at oe.tokenize (jquery-3.3.1.min.js:2)
    at oe.select (jquery-3.3.1.min.js:2)
    at Function.oe [as find] (jquery-3.3.1.min.js:2)
    at w.fn.init.find (jquery-3.3.1.min.js:2)
    at new w.fn.init (jquery-3.3.1.min.js:2)
    at w (jquery-3.3.1.min.js:2)
    at Object.<anonymous> (DashboardController.js:907)
    at Function.each (jquery-3.3.1.min.js:2)
    at Object.success (DashboardController.js:202)

**Comment by gizmocuz** [comment]
Cleared your cache?

**Comment by rwaaren** [comment]
Yes I did

**Comment by rwaaren** [comment]
Probably some glitch at the setup of my test system. Works with a fresh copy of development branch

**Comment by gizmocuz** [comment]
A cache issue

**Comment by pa3fat** [comment]
Edit: Removed because user opened a new issue #3288

**Comment by gizmocuz** [comment]
@pa3fat , have you cleared your browser cache, deleted the appcache (two different things), tried in a private browser? tried a different browser ? Asked this on the forum ? (could you specify the link)

**Comment by pa3fat** [comment]
Hi,

Cleared FF cache again minutes ago. Same behavior on edge/ie/Waterfox.

Appcache? Clearing Firefox cache (what I used) isn’t sufficient? Please advice howto will test directly.

Ron

Van: gizmocuz <notifications@github.com>
Verzonden: 30 May, 2019 10:16
Aan: domoticz/domoticz <domoticz@noreply.github.com>
CC: pa3fat <ronvandoremalen@hotmail.com>; Mention <mention@noreply.github.com>
Onderwerp: Re: [domoticz/domoticz] Beta V4.10821: Dashboard no updates on toggle scene/switch. (#3264)


@pa3fat<https://github.com/pa3fat> , have you cleared your browser cache, deleted the appcache (two different things), tried in a private browser? tried a different browser ? Asked this on the forum ? (could you specify the link)
